CIP-NFTMetadataStandard.md

```diff
@@ -0,0 +1,119 @@
+---
+CIP:
```

Comment:
20 (tentative)

Reply:
Keep in mind in your meeting that the last update proposal is the one in this comment:

https://github.com/cardano-foundation/CIPs/pull/85#discussion_r629120254

As an aside, this is also the one supported by pool.pm currently.